myia/prim/type_inferrers.py

```diff
@@ -18,6 +18,12 @@
 type_inferrer_constructors = {}
 
 
+def _shape_type(t):  # noqa: D400
```

Comment:
This should be disabled globally (dot at the end of the first line in docstrings).